microsoft-edge/webview2/release-notes.md

```diff
@@ -36,6 +36,121 @@ To load WebView2, the minimum version of Microsoft Edge or the WebView2 Runtime
 

```

Comment:
change the first two h2 headings to h4, to reduce the focus on them and thereby increase the focus on the incoming h2 sections.  This would also remove these two h2's from the Nav in upper right.  And they just look too prominent, emphasized more than we intend.

Change from h2 to h4:
Recommended browser channel and Runtime
Minimum version of the browser or Runtime to load WebView2